cellxgene_resources/curation_sample_code.ipynb

```diff
@@ -748,29 +748,21 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Extracts gene_id and gene_symbol from gtf\n",
+    "#Read in Gencode-formatted gtf file\n",
+    "genome_version = ''\n",
+    "ref_file = f's3://submissions-lattice/reference-files/refdata-gex-{genome_version}.gtf'\n",
+    "column_names = ['seqname','source','feature','start','stop','score','strand','frame','expand_me']\n",
+    "gtf_df = pd.read_table(ref_file, names=column_names, comment='#')\n",
```

Comment:
following the above suggestions, would need to edit to `my_dir + ref_file` so it can be found